extensions/wikia/Flags/Flags.setup.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * Wikia Flags Extension
+ * @author Adam Karmiński
+ *
+ */
+
+$dir = dirname(__FILE__) . '/';
+
+/**
+ * Controllers
+ */
+$wgAutoloadClasses['FlagsController'] = $dir . 'controllers/FlagsController.class.php';
+
+/**
+ * Models
+ */
+$wgAutoloadClasses['Flags\Models\FlagsModel'] = $dir . 'models/FlagsModel.class.php';
+$wgAutoloadClasses['Flags\Models\Flag'] = $dir . 'models/Flag.class.php';
+$wgAutoloadClasses['Flags\Models\FlagParameter'] = $dir . 'models/FlagParameter.class.php';
+$wgAutoloadClasses['Flags\Models\FlagType'] = $dir . 'models/FlagType.class.php';
+
+$wgExtensionCredits['other'][] = [
```

Comment:
As a general unwritten convention, the credits should go at the start of the setup file as a kind of documentation.